#base_dir: /home/hankm/catkin_ws/src/neuro_explorer
neuroexplorer:
    debug_data_save_path:   '/home/hankm/results/neuro_exploration_res' # path generates some log files
    ne_report_file:         'ne_report.txt'
    savemap_at_the_end:     true
    costmap_conf_thr:       0.4 # (0.4 for willowgarage, 0.8 for small worlds) # 0.4 (fetch)   # the higher the more strict.==> thorough exploration, go lower for faster exploration. untolerates any neighboring obstacle point   
    gridmap_conf_thr:       0.2 # (0.2 for willowgarage, 0.4 for small worlds) # the higher the more strict
    occupancy_thr:          50       # 0 ~ 100
    lethal_cost_thr:        90 #10   # 0 ~ 100  uk= -1      # the lower the more strict 
    global_width:           8192 #4096 # 8192 for WGx3 and 3072 is sufficent for WGx1 #2048 
    global_height:          8192 #4096 # 3072 is sufficient for 1x WG #2048 
    active_width:           512
    active_height:          512
    num_downsamples:        1 #2  # 1 if global with is 1024, 2 if 2048 
    weak_comp_thr:          4 #8  # 12   # 10 (willowgarage)
    unreachable_decision_bound:   0.3 # < 4 * 0.05 res >
    strict_unreachable_decision: true  # make it true if there are several obstacles in the space that blocks the robot to traverse around. true helps if there is frontier point detected beyond the wall (cannot traverse to there). Mostly "true" in real world/ noisy env, but might want to set to "false" in the simulation env where no obs in the space. 
    lambda:                 0.3     # weight for A*Net (1-lambda) goes to VizNet
    max_exploration_time:   7200     # 3600 for WG, 7200 for WGx2, and 14400 for WGx3
tf_loader:
    #fd_model_filepath:      !join [*base_dir, nn_models/fr_net/base_model] # 8 filter
    #astar_model_filepath:   !join [*base_dir, nn_models/astar_net/base_model]
    #covrew_model_filepath:  !join [*base_dir, nn_models/viz_net/base_model]
    fd_model_filepath:      /home/hankm/catkin_ws/src/neuro_explorer/nn_models/fr_net/base_model # 8 filter
    astar_model_filepath:   /home/hankm/catkin_ws/src/neuro_explorer/nn_models/astar_net/base_model
    covrew_model_filepath:  /home/hankm/catkin_ws/src/neuro_explorer/nn_models/viz_net/base_model
    cnn_height:             512
    cnn_width:              512
    num_classes:            8
